docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.8"
```

Comment:
I did go back and forth on this, too. Was hoping it was just mine because the documentation goes up through a `3.8` release, but I'm not sure either why it wasn't allowing. Will go back to `3.3`